### ISSUE TITLE
Components do not support dynamic attribute passthrough

### ISSUE THREAD

**Issue by tcorley** [eBay/ebayui-core#130]
For the example

```xml
<var myAttrs={style: "background-color: #FF0000;", "class": "my-div"} />
<ebay-breadcrumb-item href="ebay.com" ${myAttrs}>..
```
results in 

```
TypeError: Cannot read property 'name' of undefined
    at TaglibLookup.getAttribute (/Users/tcorley/Projects/search_node/node_modules/marko/compiler/taglib-lookup/TaglibLookup.js:249:29)
    at el.forEachAttribute (/Users/tcorley/Projects/search_node/node_modules/marko/compiler/ast/CustomTag.js:143:56)
    at CustomTag.forEachAttribute (/Users/tcorley/Projects/search_node/node_modules/marko/compiler/ast/HtmlElement.js:241:22)
    at buildInputProps (/Users/tcorley/Projects/search_node/node_modules/marko/compiler/ast/CustomTag.js:141:8)
    at CustomTag.generateCode (/Users/tcorley/Projects/search_node/node_modules/marko/compiler/ast/CustomTag.js:240:26)
    at Generator.generateCode (/Users/tcorley/Projects/search_node/node_modules/marko/compiler/CodeGenerator.js:260:44)
    at nodes.forEach (/Users/tcorley/Projects/search_node/node_modules/marko/compiler/CodeGenerator.js:385:22)
    at ArrayContainer.forEach (/Users/tcorley/Projects/search_node/node_modules/marko/compiler/ast/ArrayContainer.js:18:26)
    at Generator.generateStatements (/Users/tcorley/Projects/search_node/node_modules/marko/compiler/CodeGenerator.js:363:15)
    at Generator.generateBlock (/Users/tcorley/Projects/search_node/node_modules/marko/compiler/CodeGenerator.js:341:14)
    at FunctionDeclaration.generateCode (/Users/tcorley/Projects/search_node/node_modules/marko/compiler/ast/FunctionDeclaration.js:53:17)
    at Generator.generateCode (/Users/tcorley/Projects/search_node/node_modules/marko/compiler/CodeGenerator.js:260:44)
    at FunctionCall.generateCode (/Users/tcorley/Projects/search_node/node_modules/marko/compiler/ast/FunctionCall.js:47:25)
    at Generator.generateCode (/Users/tcorley/Projects/search_node/node_modules/marko/compiler/CodeGenerator.js:260:44)
    at nodes.forEach (/Users/tcorley/Projects/search_node/node_modules/marko/compiler/CodeGenerator.js:385:22)
    at Array.forEach (<anonymous>)
```

**Comment by tcorley** [comment]
Because of the difficulty of separating component attributes and pass-thru HTML ones, I'd suggest adding support for an `htmlAttributes` attribute that will explicitly pass through to the top level component.

**Comment by tcorley** [comment]
This could also pass through style (in reference to #132)

**Comment by yomed** [comment]
Thanks @tcorley, this will be fixed in `0.4.0` which will be out soon. FYI the issue with `style` is a bit different because `class` and `style` are special cases in Marko.
